src/reconcile/rec_write.c

```diff
@@ -2580,6 +2617,11 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
             WT_REF_UNLOCK(ref, previous_ref_state);
     }
 
+    if (WT_TIME_AGGREGATE_HAS_STOP(&stop_ta)) {
+        WT_RET(__wt_calloc_one(session, &mod->ta));
+        *mod->ta = stop_ta;
```

Comment:
This is non atomic, which is a problem when we read it.

Reply:
Another way to fix this is to have a flag to indicate we have finished copy.

mod->copy_ta = true;
WT_WRITE_BARRIER();
*mod->ta = stop_ta;
WT_WRITE_BARRIER();
mod->copy_ta = false;

Reply:
I don't see where the mod->ta is allocated. Can you explain?

Reply:
I changed it to a pointer assignment, so there is no need for new variable.